### ISSUE TITLE
Page date and time zone.

### ISSUE THREAD

**Issue by Narno** [PHPoole/PHPoole-library#94]
Page::getDate() should use a specific time zone, ans this time zone value should be available in config.

`$date = new \DateTime($date, new \DateTimeZone($timezone));`

ie:
```
$datetime = new \DateTime();
$datetime->setTimestamp($date);
$datetime->setTimezone(new \DateTimeZone('Europe/Paris'));
return $datetime->format(\DateTime::W3C);
```

**Comment by Narno** [comment]
Or just use dedicated Twig filter :-)

http://twig.sensiolabs.org/doc/filters/date.html#timezone

**Comment by Narno** [comment]
Fixed by 42bc380bd159f69fd67a9a01d42c885f4d806d91.
